### ISSUE TITLE
False positive for Lint/MixedCaseRange

### ISSUE THREAD

**Issue by ushi-as** [rubocop/rubocop#11986]
Hello,

i found a false positive for the `Lint/MixedCaseRange` cop.  `'a'..'zz'` is reported as mixed case range, which is obviously not.

--------

## Expected behavior

It reports no offense.

## Actual behavior

It reports `'a'..'zz'` as mixed case range.

```
W: Lint/MixedCaseRange: Ranges from upper to lower case ASCII letters may include unintended characters. Instead of A-z (which also includes several symbols) specify each range individually: A-Za-z and individually specify any symbols.
      RANGE = ('a'..'zz')
               ^^^^^^^^^
```

## Steps to reproduce the problem

Let rubocop inspect code which includes `'a'..'zz'`.

## RuboCop version

1.53.0
